src/main/kotlin/com/github/mitchellolsthoorn/testgenie/evosuite/EvoSuiteRunner.kt

```diff
@@ -38,7 +44,31 @@ class EvoSuiteRunner {
                 "-class", classFQN, // class FQN inside the project classpath of the class we're generating tests for
                 "-projectCP", projectClassPath, // class path of the project we're generating tests for
                 "-Djunit_tests=true", // disable writing to 'evosuite-tests' in working directory
-                "-Dnew_statistics=false" //disable writing to 'evosuite-report' in working directory
+                "-Dnew_statistics=false", //disable writing to 'evosuite-report' in working directory
+
+                // Parameters from tool window
+                "-Dsearch_budget=${toolWindowState!!.searchBudget}",
+                "-Dlocal_search_budget=${toolWindowState.localSearchBudgetValue}",
+                "-Dlocal_search_budget_type=${toolWindowState.localSearchBudgetType}",
+                "-Dstopping_condition=${toolWindowState.stoppingCondition}",
```

Comment:
In my opinion you need to change this line to:
`"-Dstopping_condition=${toolWindowState.stoppingCondition.name}",`
Because right now you are getting the display value of the enum, rather than its name.

Reply:
Well spotted! I have changed all 3 enum parameters (localSearchBudgetType, populationLimit, stoppingCriteria)